packages/wonder-blocks-dropdown/components/multi-select.md

```diff
@@ -329,7 +330,10 @@ import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
     <MultiSelect
         aria-labelledby="label-for-multi-select"
         id="unique-multi-select"
-        placeholder="Accessible MultiSelect"
+        labels={{
+            noneSelected: "Accessible MultiSelect",
+            someSelected: (numSelectedValues) => `${numSelectedValues} planets`,
+        }}
```

Comment:
Do we no longer support `placeholder`?  If so, do the flow types need to change?

Reply:
I removed the `placeholder` prop as part of this PR: https://github.com/Khan/wonder-blocks/pull/659, and replaced it with `labels.noneSelected`, but it seems that I forgot to update all the code examples written in the markdown file.